plugins/opsgenie/icon_opsgenie/actions/get_alert/schema.py

```diff
@@ -0,0 +1,79 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Retrieve alert from OpsGenie"
+
+
+class Input:
+    IDENTIFIER = "identifier"
+    IDENTIFIERTYPE = "identifierType"
+    
+
+class Output:
+    DATA = "data"
+    REQUESTID = "requestId"
+    
+
+class GetAlertInput(insightconnect_plugin_runtime.Input):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "identifier": {
+      "type": "string",
+      "title": "Alert ID",
```

Comment:
This can be Alert ID, tinyID or Alias
```suggestion
      "title": "Alert ID",
```

Reply:
Ok, updated